From 274f163e88b72a2dafbc946c1648731aaec32427 Mon Sep 17 00:00:00 2001
From: Drew Moseley <drew.moseley@northern.tech>
Date: Mon, 9 Sep 2019 20:09:44 +0000
Subject: [PATCH] Set wifi_module to brcm since that's what I have

Signed-off-by: Drew Moseley <drew.moseley@northern.tech>
---
 include/configs/pico-imx7d.h  | 2 +-
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/pico-imx7d.h b/include/configs/pico-imx7d.h
index 251db46ec9..f924d80846 100644
--- a/include/configs/pico-imx7d.h
+++ b/include/configs/pico-imx7d.h
@@ -184,7 +184,7 @@
 	"som=imx7d\0" \
 	"baseboard=pi\0" \
 	"form=pico\0" \
-	"wifi_module=qca\0" \
+	"wifi_module=brcm\0" \
 	"default_baseboard=pi\0" \
 	"fdt_high=0xffffffff\0" \
 	"initrd_high=0xffffffff\0" \
-- 
2.23.0

